lib/state/state_container.dart

```diff
@@ -43,6 +43,12 @@ class StateContainerState extends State<StateContainer> {
         ..isLoading = state.isLoading);
     });
   }
+
+  void currentTheme(String t) {
```

Comment:
since we are setting the currentTheme, better to name this as setCurrentTheme. Or make this a set method